app/src/main/java/org/sopt/and/component/ContentListTitle.kt

```diff
@@ -0,0 +1,51 @@
+package org.sopt.and.component
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import org.sopt.and.ui.theme.pretendardFamily
+
+@Composable
+fun ContentListTitle(
+    title: String,
+    button: Int?,
+    navigateTo: () -> Unit
```

Comment:
해당 함수를 사용하는 곳이 없는 것 같은데 필수로 사용해야하듯이 되어 있어요! 없애거나 default value로 nullable 하게 만드는건 어떨까요? 사용할 때는 navigateTo?.invoke() 이렇게 쓰면 됩니다!
```suggestion
    navigateTo: (() -> Unit)?
```

Reply:
오 네 참고해서 적용해보겠습니다!!